datapackage.json

```diff
@@ -0,0 +1,147 @@
+{
+  "name": "corruption-perceptions-index",
+  "resources": [
+    {
+      "path": "data/cpi.csv",
+      "name": "data",
+      "schema": {
+        "fields": [
+          {
+            "description": "",
+            "format": "default",
```

Comment:
Same here - why do we specify format to default? it will automatically be done right?